src/sequences/SimpleFactor.ts

```diff
@@ -0,0 +1,219 @@
+/**
+ * Provides a utility function so that front-end sequences can factor their
+ * smallish values; should go away once essentially all sequences are
+ * computed in the backend.
+ */
+
+// Here's a list of enough primes to make sure we can factor every number
+// up to a million. [Sadly, prettier insists on just one bigint per line :-/ ]
+const smallPrimes = [
```

Comment:
I think an ignore directive is appropriate here.

Adding `// prettier-ignore` above `const smallPrimes = [` allows you to have the array on multiple lines like it was in `SequenceFormula.ts` before this PR.

Reply:
OK, when we settle the other item, I will add the ignore directive and condense the list.